public/videoStyles.css

```diff
@@ -0,0 +1,316 @@
+
```

Comment:
Maybe we should throw a comment at the top that says this whole file comes from the rh5v or whatever it's called library